cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/resource/KafkaRoller.java

```diff
@@ -657,6 +662,17 @@ private void checkIfRestartOrReconfigureRequired(NodeRef nodeRef, boolean isCont
         restartContext.brokerConfigDiff = brokerConfigDiff;
     }
 
+    /**
+     * Returns the information about the features within the Kafka Cluster
+     * @return information about the features
+     */
+    /* test */ FeatureMetadata featureMetadata() throws ForceableProblem, InterruptedException {
```

Comment:
This seems pretty risky given how we don't know what state the cluster is in. Or?

Reply:
Wdym? Can you elaborate a bit?

Reply:
Anything to do with rolling is sensitive to whether you can or cannot connect to the cluster. So adding yet another connection there seems pretty risky to me.

Reply:
Ok, it might work in that case I guess.